### ISSUE TITLE
Not working with prefix

### ISSUE THREAD

**Issue by cvalentin-dkt** [compilerla/conventional-pre-commit#91]
Hello,

I try to set hook but it's doesn't detect commit prefix :/

`.pre-commit.config.yaml` : 

```yaml
minimum_pre_commit_version: 3.6.0
fail_fast: true
repos:

  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v4.5.0
    hooks:
      - id: trailing-whitespace # Trims trailing whitespace
        stages: [pre-commit]
      - id: check-yaml # Validates YAML files
        stages: [pre-commit]
        exclude: ^src/docapi/.*|^tests/templates/.*|^src/templates/.*|
      - id: check-json # Validates JSON files
        stages: [pre-commit]
        exclude: ^src/docapi/.*|^tests/templates/.*|^src/templates/.*|
      - id: check-added-large-files # Checks for files that are added to the repository that are larger than a threshold
        stages: [pre-commit]
      - id: check-case-conflict # Checks for files that would conflict in case-insensitive filesystems
        stages: [pre-commit]
      - id: check-merge-conflict # Checks for files that contain merge conflict strings
        stages: [pre-commit]
      - id: detect-private-key # Check for the existence of private keys
        stages: [pre-commit]
      - id: check-executables-have-shebangs # Checks that executables have shebangs
        stages: [pre-commit]
      - id: end-of-file-fixer  # Makes sure files end in a newline and only a newline
        stages: [pre-commit]

  - repo: local
    hooks:
      - id: flake8
        stages: [pre-commit]
        name: flake8
        always_run: true
        entry: bash -c 'make flake8'
        language: system
        types: [python]
        pass_filenames: false

      - id: black
        stages: [pre-commit]
        name: black
        always_run: true
        entry: bash -c 'make black'
        language: system
        types: [python]
        pass_filenames: false

      - id: pylint
        stages: [pre-commit]
        name: pylint
        always_run: true
        entry: bash -c 'make pylint'
        language: system
        types: [python]
        pass_filenames: false

  - repo: https://github.com/compilerla/conventional-pre-commit
    rev: v3.1.0
    hooks:
      - id: conventional-pre-commit
        stages: [commit-msg]
        args: [--strict, --force-scope, core, build, chore, ci, docs, feat, fix, perf, style, refactor, test]
```

Result with command : `git add --all; git commit -m "fix: test"`

![image](https://github.com/compilerla/conventional-pre-commit/assets/81801129/f6c99f59-8d01-441b-860f-0a938b7b8ec9)

Pre-Commit version : **3.6.1**

**Comment by bristi** [comment]
You are using `--force-scope` but you don't seem to be using a scope in your commit message? Eg `fix(backend): test` where backend is the scope.

**Comment by thekaveman** [comment]
Thanks @bristi! You are correct, the `--force-scope` flag here is what is causing the validation failure.

**Comment by cvalentin-dkt** [comment]
Thx all :)
